packages/Confirmation/src/Confirmation.js

```diff
@@ -77,7 +77,7 @@ const Confirmation = props => {
     if (isConfirmOpen) {
       setIsConfirmOpen(false);
       if (triggerRef.current) triggerRef.current.focus();
-      setTimeout(onClose, 0);
+      setTimeout(onClose, 250);
```

Comment:
Added a delay to accommodate the animation of the confirmation closing. In the case of the dropdown Component (which uses confirmation component) we can then use this callback to close the menu.

Reply:
Maybe adding the explanation in the code in case we forget it later on?

Reply:
Yeah ok i shall, although i'm not keen on leaving comments in the code in general